### PR TITLE
NW-6 | AREEB-SATTAR | REACT| High-Score-Tables| WEEK-1

### PR DESCRIPTION
**Pull request by areebsattar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>High Scores Table</title>
```
Comment (SpectacularLandscape) [inline index.html:8]: Good catch changing this title, it's often forgotten 😉

File: src/Table.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import allCountryScores from "./scores";
+import "./Table.css"
+
+function Table() {
```
Comment (SpectacularLandscape) [inline src/Table.jsx:5]: Is there a more descriptive name you could use for this component?

File: src/Table.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          return (
+            <div key={index}>
+              <table>
+                <thead>
+                  <tr>
+                    <th scope="col">{name}</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {scores.map(({ n, s }, index) => {
```
Comment (SpectacularLandscape) [inline src/Table.jsx:20]: Great use of maps! I think we can use some more descriptive variable names here however, which introduces a little destructuring trick.

Personally I don't like the variable names `n` and `s`, so if I want to, I can rename `n` as `personName` and `s` as `score` using this syntax:

```js
{scores.map(({n : personName, s : score }, index) => {
    // map logic here
    console.log(`A person with the name '${personName}' has a score of ${score}`)
})}
```

Comment (SpectacularLandscape) [inline src/Table.jsx:20]: Where we have a nested `map`, I would consider a separate component. As a rule of thumb, if we have another loop inside of a component, it's probably a good opportunity to refactor into multiple components

File: src/Table.jsx (hunk 2 above)
Comment (areebsattar) [reply src/Table.jsx:5]: Possibly the full name of the ticket I guess

File: src/Table.jsx (hunk 3 above)
Comment (areebsattar) [reply src/Table.jsx:20]: I will keep this in mind for future

Comment (areebsattar) [reply src/Table.jsx:20]: Sounds right, I will try to do this in future.

**Pull request merged**